hamnosys_dicts.py

```diff
@@ -0,0 +1,238 @@
+DescriptionDict = {
+    "1": "name",
+    "2": "start",
+    "3": "end",
+    "4": "Symmetry operator",
+    "5": "Dom hand - Handshape - base form",
+    "6": "Dom hand - Handhape - Thumb position",
+    "7": "Dom hand - Handshape - Bending",
+    "8": "Dom hand - Handposition - Extended finger direction",
+    "9": "Dom hand - Handposition - Palm orientation",
+    "10": "Dom hand - Handlocation - Frontal plane l/r",
+    "11": "Dom hand - Handlocation - Frontal plane t/b",
+    "12": "Dom hand - Handlocation - Distance from the body",
+}
+
+HandlocationFronalPlaneRL = {
+    "0": "",  # left to the left
+    "1": "",  # left
+    # "2" : "", #center
+    "3": "",  # right
+    "4": "",  # right to the right
+}
+
+HandLocationFronalPlaneTB = {
+    #   "0" : None,
```

Comment:
also here

Reply:
It is here to be used more like a legend